tests/cluecode/data/plugin_filter_clues/filtered-expected2.json

```diff
@@ -77,9 +77,11 @@
           }
         }
       ],
+      "unknown_licenses": [],
```

Comment:
unknown_licenses are less important than licenses so they should come after always